orca-clouddriver/src/main/java/com/amazon/spinnaker/orca/aws/lambda/utils/LambdaCloudDriverUtils.java

```diff
@@ -0,0 +1,456 @@
+ *  * distributed under the License is distributed on an "AS IS" BASIS,
+ *  * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  * See the License for the specific language governing permissions and
+ *  * limitations under the License.
+ *
+ *
+ */
+
+package com.amazon.spinnaker.orca.aws.lambda.utils;
+
+import com.amazon.spinnaker.orca.aws.lambda.CloudDriverProperties;
+import com.amazon.spinnaker.orca.aws.lambda.model.LambdaDefinition;
+import com.amazon.spinnaker.orca.aws.lambda.traffic.model.LambdaCloudDriverInvokeOperationResults;
+import com.amazon.spinnaker.orca.aws.lambda.traffic.model.LambdaPipelineArtifact;
+import com.amazon.spinnaker.orca.aws.lambda.upsert.model.LambdaDeploymentInput;
+import com.amazon.spinnaker.orca.aws.lambda.verify.model.LambdaCloudDriverErrorObject;
+import com.amazon.spinnaker.orca.aws.lambda.verify.model.LambdaCloudDriverResultObject;
+import com.amazon.spinnaker.orca.aws.lambda.verify.model.LambdaCloudDriverTaskResults;
+import com.amazon.spinnaker.orca.aws.lambda.verify.model.LambdaVerificationStatusOutput;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.DeserializationFeature;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.node.ArrayNode;
+import com.fasterxml.jackson.databind.node.IntNode;
+import com.fasterxml.jackson.databind.node.TextNode;
+import com.fasterxml.jackson.databind.type.TypeFactory;
+import com.google.common.io.CharStreams;
+import com.netflix.spinnaker.kork.artifacts.model.Artifact;
+import com.netflix.spinnaker.kork.core.RetrySupport;
+import com.netflix.spinnaker.orca.api.pipeline.models.StageExecution;
+import com.netflix.spinnaker.security.AuthenticatedRequest;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.util.Comparator;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+import okhttp3.*;
+import org.apache.commons.lang3.ObjectUtils;
+import org.apache.commons.lang3.math.NumberUtils;
+import org.pf4j.util.StringUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+@Component
+public class LambdaCloudDriverUtils {
+  private static final Logger logger = LoggerFactory.getLogger(LambdaCloudDriverUtils.class);
+  private static final ObjectMapper objectMapper = new ObjectMapper();
+  private static final String CLOUDDRIVER_GET_PATH = "/functions";
+
+  static {
+    objectMapper.configure(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES, false);
+    objectMapper.configure(DeserializationFeature.ACCEPT_SINGLE_VALUE_AS_ARRAY, true);
+  }
+
+  @Autowired CloudDriverProperties props;
+
+  public LambdaCloudDriverResponse postToCloudDriver(String endPointUrl, String jsonString) {
```

Comment:
This is doing what Kato is supposed to be doing 🤔 Would hope that we don't need to do things differently